x-pack/plugins/security_solution/server/lib/detection_engine/signals/types.ts

```diff
@@ -166,6 +168,12 @@ export interface GetResponse {
   _source: SearchTypes;
 }
 
+export type EventHit = Exclude<TypeOfFieldMap<EcsFieldMap>, '@timestamp'> & {
+  '@timestamp': string;
```

Comment:
interesting, are we removing `@timestamp` to add it back? was it not a string type before?

Reply:
@ecezalp `@timestamp` isn't required in `SimpleHit` so it makes things a bit weird... here, `@timestamp` is guaranteed to be set.